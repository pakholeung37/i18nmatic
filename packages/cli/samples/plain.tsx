const ERROR_MESSAGE = "请稍后再试"
const review = {
  id: 1,
  name: "测试",
}
const reviews = [
  {
    id: 1,
    name: "张先生",
  },
  {
    id: 2,
    name: "李女士",
  },
]

const typeNameMap: Record<string, string> = {
  Folder: "文件夹",
  FolderExpanded: "文件夹",
  Model: "持久模型",
  ModelView: "普通模型",
  Statistic: "统计模型",
  Event: "业务事件",
  Action: "扩展服务",
  ServiceConfig: "业务配置",
  Scene: "页面场景",
  ServiceDefinition: "编排服务",
  Flow: "业务流",
  Trigger: "触发器监听",
  EventDefinition: "触发器定义",
  Process: "业务流",
  OverviewProcess: "业务流程",
  WorkflowGroup: "审批流场景",
  NoticeScene: "通知场景",
  PrintScene: "打印场景",
  View: "视图",
  MenuTree: "菜单树",
  TakeCode: "取号中心",
  Connector: "连接器",
  Menu: "菜单",
  Module: "模块",
  StateConfig: "状态配置",
  Validation: "校验规则",
  ImportExportTemplate: "导入导出场景",
  Permission: "功能权限项",
  DataCondition: "数据权限规则",
  DataControlDimension: "数据控权维度",
  AIAgent: "智能体",
  KnowledgeBase: "知识库",
}

const REQUEST_ERROR_STATUS: Record<number, string> = {
  0: '网络异常，请检查网络连接',
  200: '服务器成功返回请求的数据。',
  201: '新建或修改数据成功。',
  202: '一个请求已经进入后台排队（异步任务）。',
  204: '删除数据成功。',
  400: '发出的请求有错误，服务器没有进行新建或修改数据的操作。',
  401: '用户没有权限（令牌、用户名、密码错误）。',
  403: '用户得到授权，但是访问是被禁止的。',
  404: '发出的请求针对的是不存在的记录，服务器没有进行操作。',
  405: '请求方法不被允许。',
  406: '请求的格式不可得。',
  410: '请求的资源被永久删除，且不会再得到的。',
  422: '当创建一个对象时，发生一个验证错误。',
  500: '服务器发生错误，请检查服务器。',
  502: '网关错误。',
  503: '服务不可用，服务器暂时过载或维护。',
  504: '网关超时。',
}
