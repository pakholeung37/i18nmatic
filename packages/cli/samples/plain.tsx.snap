import { t } from "@terminus/t-console-framework"
const ERROR_MESSAGE = t("请稍后再试")
const review = {
  id: 1,
  name: t("测试"),
}
const reviews = [
  {
    id: 1,
    name: t("张先生"),
  },
  {
    id: 2,
    name: t("李女士"),
  },
]
const typeNameMap: Record<string, string> = {
  Folder: t("文件夹"),
  FolderExpanded: t("文件夹"),
  Model: t("持久模型"),
  ModelView: t("普通模型"),
  Statistic: t("统计模型"),
  Event: t("业务事件"),
  Action: t("扩展服务"),
  ServiceConfig: t("业务配置"),
  Scene: t("页面场景"),
  ServiceDefinition: t("编排服务"),
  Flow: t("业务流"),
  Trigger: t("触发器监听"),
  EventDefinition: t("触发器定义"),
  Process: t("业务流"),
  OverviewProcess: t("业务流程"),
  WorkflowGroup: t("审批流场景"),
  NoticeScene: t("通知场景"),
  PrintScene: t("打印场景"),
  View: t("视图"),
  MenuTree: t("菜单树"),
  TakeCode: t("取号中心"),
  Connector: t("连接器"),
  Menu: t("菜单"),
  Module: t("模块"),
  StateConfig: t("状态配置"),
  Validation: t("校验规则"),
  ImportExportTemplate: t("导入导出场景"),
  Permission: t("功能权限项"),
  DataCondition: t("数据权限规则"),
  DataControlDimension: t("数据控权维度"),
  AIAgent: t("智能体"),
  KnowledgeBase: t("知识库"),
}
const REQUEST_ERROR_STATUS: Record<number, string> = {
  0: t("网络异常，请检查网络连接"),
  200: t("服务器成功返回请求的数据。"),
  201: t("新建或修改数据成功。"),
  202: t("一个请求已经进入后台排队（异步任务）。"),
  204: t("删除数据成功。"),
  400: t("发出的请求有错误，服务器没有进行新建或修改数据的操作。"),
  401: t("用户没有权限（令牌、用户名、密码错误）。"),
  403: t("用户得到授权，但是访问是被禁止的。"),
  404: t("发出的请求针对的是不存在的记录，服务器没有进行操作。"),
  405: t("请求方法不被允许。"),
  406: t("请求的格式不可得。"),
  410: t("请求的资源被永久删除，且不会再得到的。"),
  422: t("当创建一个对象时，发生一个验证错误。"),
  500: t("服务器发生错误，请检查服务器。"),
  502: t("网关错误。"),
  503: t("服务不可用，服务器暂时过载或维护。"),
  504: t("网关超时。"),
}
